src/app/components/steps/tidsrom/OpplysningerOmTidsromStep.tsx

```diff
@@ -19,6 +19,7 @@ import { AppFormField, PleiepengesøknadFormData } from '../../../types/Pleiepen
 import { Feature, isFeatureEnabled } from '../../../utils/featureToggleUtils';
 import { brukerSkalBekrefteOmsorgForBarnet, brukerSkalBeskriveOmsorgForBarnet } from '../../../utils/tidsromUtils';
 import {
+    getMinDate,
```

Comment:
dayjs har egen min funksjon, så du kan kanskje bruke den i stedet for å lage en egen? impoerter minMax fra dayjs/plugins/minMax